trigger: log error in triggers that does not throw an error (#9743)
tarantool/tarantool: changelogs/unreleased/gh-9309-errors-in-triggers.md

## feature/box

> * Now all triggers are either have a direct impact on the execution flow
>   (e.g. close a connection or throw an error) or print a message to error
>   log when they throw an error (gh-9309).

Reviewer: ```suggestion
* Now all triggers either have a direct impact on the execution flow
  (for example, closing a connection or throwing an error) or print a message to the error
  log when they throw an error (gh-9309).
```
